doc/source/autodoc.rst

```diff
@@ -45,6 +45,11 @@ For the automatic extraction to work, the code must be particularly formatted. S
   as the ``type`` keyword, and individual values must each have their own line
 - record-defined types must have the ``record`` keyword on the same line as the
   ``type`` keyword, and each element of that record must be on its own line
+- it's possible to create groups of ports and generics, but there are some rules,
+  that must be observed
+- every group must start and end with at least two ``=``, then name of group and
+  description folows
+- behind the block that indicates group, folows list of ports or generics
```

Comment:
- it's possible to create groups of ports and generics, but there are some rules that must be observed 
  
- every group must start and end with at least two ``=`` symbols, followed by the name of the group and the group's description
  
- underneath, the list of ports or generics should follow